### PR TITLE
Create, export and import paper wallet

### PR DESCRIPTION
**Pull request by cryptofuture**
- Save account UTC/JSON as QR code in PNG
- Import account as PNG QR code in UTC/JSON

### CODE REVIEW

File: interface/elements/msc-profile-view/msc-profile-view.js [hunk 1]
```diff
@@ -250,6 +253,88 @@ Polymer({
...
+      } else if (platform.includes("linux")) { //linux
+        var pathOfKey = process.env.HOME + '/.musicoin/keystore/UTC--';
+      }
+    png = new (require('pngjs').PNG)();
+    document.getElementById('fileDialogPaperImport').click();
+    document.querySelector('#fileDialogPaperImport').addEventListener("change", function() {
+    var PaperWallet = document.getElementById('fileDialogPaperImport').value;
+    png.parse(fs.readFileSync(PaperWallet), function(err, decodedPng) {
+    if(err) {
+      //console.error(err);
```
Comment (Varunram) [inline interface/elements/msc-profile-view/msc-profile-view.js]: Maybe we should exit here? We don't want stuff falling through if there is an error

**Comment by Varunram** [comment]
Also, a slight inconvenience is that if there is already an address present, importing the same qr seems to create two copies of it. Maybe we should check if the address already exists before actually displaying it along with the other addresses.

**Comment by cryptofuture** [comment]
>Also, a slight inconvenience is that if there is already an address present, importing the same qr seems to create two copies of it. Maybe we should check if the address already exists before actually displaying it along with the other addresses.
Same apply to UTC/JSON files, mnemonic imports and private keys. If account is like "bad" (person don't remember password to it), person still able to import another UTC/JSON file or private key, which might use another password, and still be usable.
Though this creates problem with deletion - it stops working. But right now I consider this a feature, since removing two or more UTC/JSON files with same account for one go - is definitely not what anyone want.

**Comment by Varunram** [comment]
An unintended redundancy solution hehe. Sounds good.

**Pull request merged**